erpnext/stock/doctype/pick_list/pick_list.py

```diff
@@ -250,6 +250,7 @@ def create_stock_entry(self):
 				if location.package_tag:
 					stock_entry = update_stock_entry_items_for_target(location, stock_entry)
 			stock_entry.set_stock_entry_type()
+			stock_entry.validate_item()
```

Comment:
The `validate_item` function is automatically called when the stock entry is saved on line 257 because validate_item is in validate of stock_entry. Does this change even fix the issue? coz ideally, the function was being called earlier as well.

Reply:
Yes, the function validate_item is called automatically when the stock entry is saved on line 257, 
but before that set_incoming_rate function gets called on line 254. in that function, items are fetched to set the incoming rate.
If the item missing any parameter or value then it throws the error. Therefore the validate_item function is required before set_incoming_rate.